core/server/index.js

```diff
@@ -72,7 +72,11 @@ function init() {
         parentApp = require('./app')();
 
         debug('Express Apps done');
-
+    }).then(function () {
+        return auth.validation.switch({
```

Comment:
Actually you are right. 
I didn't do it, because `auth.init` runs in background, but it can still run in background after the validation. Will update 👍